_features/discoverable.html

```diff
@@ -2,34 +2,22 @@
 title: Discoverable
 slug: discoverable
 description: Are identifiable as applications and search engines can find them.
+usecases:
+- <p>Progressive Web Applications offer unique capabilities; as a developer, I want
+  my web applications to be promoted in search engines when the user is using
+  modern platforms.</p>
+- <p>As a search engine owner I want to highlight those applications with best UX
+  promoting them to the upper positions.</p>
+- <p>As a browser developer I want to give apps special capabilities and allow
+  them to <em>escape the tab</em>.</p>
+- <p>As a marketplace owner I want to gather information about progressive web
+  application to rating and sorting them.</p>
```

Comment:
"Marketplace" is a bit ambiguous here. I would prefix it with "app," i.e. call it an "app marketplace." Note also that the [Wikipedia entry for the concept](https://en.wikipedia.org/wiki/App_store) uses the phrase "app store" as the primary name for the concept, although it does reference "app marketplace" as an alternative.

Also, nits:
* application -> applications (or, preferably, "apps")
* rating and sorting -> rate and sort (although I would probably use a word like "categorize" or "organize" here)